src/sql/workbench/services/accountManagement/browser/accountDialog.ts

```diff
@@ -193,9 +195,39 @@ export class AccountDialog extends Modal {
 				for (const addedProvider of addedProviders) {
 					this.addProvider(addedProvider);
 				}
+			})
+			.then(async () => {
+				await this.addGitHubCopilotProvider();
 			});
 	}
 
+	private async addGitHubCopilotProvider(): Promise<void> {
```

Comment:
It would be great to make this generic for all auth providers - the account dialog shouldn't really know anything about specific implementations. What you have here is basically already doing that - but you could finish the job by removing the `providerId === 'github'` check and then using the ID directly along with the session.account.label for the displayName

(of course having two separate "account" services is also not very ideal, but that'd be a bigger refactor. Something to think about @cheenamalhotra @kisantia though as a way to standardize more of our stuff with VS Code)